io/BUILD.go

```diff
@@ -0,0 +1,15 @@
+package io
+
+import (
+	"libsupcxx/RULES/libsupcxx/lib"
+)
+
+var Lib = lib.Library{
```

Comment:
Can't you use the default cc.Lib here? I always try to use the generic build rule whenever possible and we don't need the `OnlyForConfig` here.

Reply:
I agree with sticking to cc.Lib if possible. Will change.

Reply:
Also, I would call the variable 'Io' not 'Lib'.

Reply:
> Can't you use the default cc.Lib here? I always try to use the generic build rule whenever possible and we don't need the `OnlyForConfig` here.

On second thought, I need lib.Library to get the correct include path.